engine/src/mac-core.mm

```diff
@@ -2056,7 +2056,7 @@ int platform_main(int argc, char *argv[], char *envp[])
 	MCStringRef *t_new_envp;
 	/* UNCHECKED */ MCMemoryNewArray(1, t_new_envp);
 	
-	int i = 0;
+	uindex_t i = 0;
```

Comment:
I'm confused.  `i` is used in this function in the loop:

```c++
for (i = 0; i < argc; i++)
    MCValueRelease(t_new_argv[i]);
```

Now, `argc` is signed, so surely that introduces a signed-unsigned comparison?  Am I missing something?

I suggest introducing new separate loop variables, e.g. `t_env_num` and `t_arg_num`.